mungegithub/mungers/submit-queue.go

```diff
@@ -381,8 +383,8 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 			gcs = utils.NewTestUtils("bucket", "logs", overrideUrl)
 		} else {
 			gcs = utils.NewWithPresubmitDetection(
-				utils.KubekinsBucket, utils.LogDir,
-				utils.PullKey, utils.PullLogDir,
+				sq.features.GCSInfo.BucketName, sq.features.GCSInfo.LogDir,
+				sq.features.GCSInfo.PullKey, sq.features.GCSInfo.PullLogDir,
```

Comment:
I had a discussion with @lavalamp about this. GCSInfo needs to be used by two different mungers, and supplied from the commandline. Because it could be used by several mungers, it should exist outside any individual munger in my opinion. The options were:
* to make them global flags populated when mungegithub is run, in the main function (but that makes them sort of opaque).
* Introduce a new struct, which would in turn be passed into every munger in its Initialize() method (this needed changes to the Mungers interface).
* Making it a feature, similar to the feature 'gitrepos' used to read the locally fetched repository in blunderbuss.

I went with the third because it seemed like the most natural way to approach it, and its similarity to how 'gitrepos' works. If there is a better mechanism I may have missed, please let me know. :)